docs/node-mixin/dashboards/node.libsonnet

```diff
@@ -115,9 +115,9 @@ local gauge = promgrafonnet.gauge;
         |||
           100 -
           (
-            sum(node_filesystem_avail_bytes{%(nodeExporterSelector)s, %(fsSelector)s, instance="$instance"}
+            sum(node_filesystem_avail_bytes{%(nodeExporterSelector)s, %(fsSelector)s, instance="$instance"})
           /
-            sum(node_filesystem_size_bytes{%(nodeExporterSelector)s, %(fsSelector)s, instance="$instance"}
+            sum(node_filesystem_size_bytes{%(nodeExporterSelector)s, %(fsSelector)s, instance="$instance"})
```

Comment:
How about moving the `%(fsSelector)s` part to the end? Then it should still work with an empty string. I would like that more than the bogus value in this P.

Reply:
Which has to be done for all instances of fsSelector and diskDeviceSelector

Reply:
That's indeed much better :+1: